Products/ZenModel/migrate/data/rabbitmq-6.0.0.conf

```diff
@@ -0,0 +1,40 @@
+#
+# This filter should be used for all RabbitMQ operational logs (rabbit@rbt[0-9]*.log - rabbit@rbt[0-9]*-sasl.log)
+#
+
+# If it doesn't start with a "=" then it is part of the same log message
+multiline {
+    pattern => "^="
+    negate => true
+    what => "previous"
+}
+
+grok {
+    # Match this line of the form: "=INFO REPORT==== 2-Aug-2017::14:30:35 ==="
+    match => { "message" => "^=%{WORD:loglevel} REPORT==== %{MONTHDAY:day}-%{MONTH:month}-%{YEAR:year}::%{TIME:time} ===.*$" }
+}
+
+# interpret message header text as "severity"
+mutate {
+    gsub => [
```

Comment:
Needs to match any changes made for https://github.com/zenoss/zenoss-service/pull/442